src/docfx/build/metadata/SystemMetadata.cs

```diff
@@ -31,6 +31,8 @@ internal class SystemMetadata
         [JsonProperty("_op_gitContributorInformation")]
         public ContributionInfo? ContributionInfo { get; set; }
 
+        public string?[]? GitContributorInformation { get; set; }
```

Comment:
Use `string[]?` as the type, each array item must not be null.

Reply:
The type of "Name" property in `Contributor.cs` is `string?`, thus we have to use `string?[]?` here